lib/bridge/pbn.rb

```diff
@@ -1,7 +1,5 @@
+require_relative('../bridge')
```

Comment:
You shouldn't need to do `require_relative` since in a gem the `lib` directory is added to the namespace. `require 'bridge'` should work just fine.